pytket/tests/conftest.py

```diff
@@ -1,72 +0,0 @@
-# Copyright 2019-2023 Cambridge Quantum Computing
```

Comment:
Yes, they were not used anywhere and also causing type errors. I removed them and all the tests were still fine so thought it was just never cleaned up maybe

Reply:
Hmm, the functions `add_randomly_*_qubit_gates()` don't seem to be used, as you say. Looks like `circuit`, `circuit_4` and `optype` are used in the file `tests/timing_circuit.py` but this is not run by default (presumably because the filename doesn't end with `test`) and may be itself obsolete...

Reply:
Also happy to just remove `timing_circuit.py`; it looks like an old file that has no purpose anymore.